curl-helper.c

```diff
@@ -3766,6 +3777,21 @@ CURLOptionMapping implementedOptionMap[] =
 #else
   HAVENOT(AWS_SIGV4),
 #endif
+#if HAVE_DECL_CURLOPT_TCP_KEEPALIVE
```

Comment:
there is no need for configure test btw, these options were added in 7.25.0, ocurl requires 7.28.0 at minimum, can just add unconditionally.